zmsbackend/src/Zmsbackend/Helper/QueueProcessListToScopeAdmin.php

```
@@ -43,8 +43,8 @@ public function startProcessing($commit): void
                    ->withSortedArrival()
                    ->toProcessList();
                 if ($processList->count() > 0) {
-                    if ($this->sendListToQueue($scope, $processList) && $this->verbose) {
-                        \App::$log->info('Send processList to scope admin', [
+                    if ($this->writeListToQueue($scope, $processList) && $this->verbose) {
```

PHPStan (2.2.7)

[error] Result of method BO\Zmsbackend\Helper\QueueProcessListToScopeAdmin::writeListToQueue() (void) is used.


(method.void)